gbfs.md

```diff
@@ -450,15 +446,15 @@ Example:
 
 ### free_bike_status.json
 
-Describes vehicles that are not at a station and are not currently in the middle of an active ride.
+Describes all vehicles that are not currently in active rental. Required for free floating (dockless) vehicles. Conditionally required for station based (docked) vehicles when [vehicle_types.json](#vehicle_typesjson) has been defined. Optional for station based (docked) vehicles when [vehicle_types.json](#vehicle_typesjson) has not been defined. Vehicles that are part of an active rental must not appear in this feed.
```

Comment:
Same question here and for station_id: Why are these conditioned on vehicle_types feed being present? I would just say that it is "optional for station based vehicles". I think having vehicle_types feed (i.e. being able to describe what type each vehicle is) is independent from being able to list just list vehicles at a station.